logbook-json/src/main/java/org/zalando/logbook/json/JsonCompactor.java

```diff
@@ -19,6 +20,7 @@ boolean isCompacted(final String json) {
         return json.indexOf('\n') == -1;
     }
 
+    // TODO json.replace("\n", "") would cover our biggest use case
```

Comment:
Do you mean as an alternative more light-weight version?

Reply:
Yes. Maybe making JsonCompactor an interface and offer two implementations.